packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md

```diff
@@ -0,0 +1,263 @@
+---
+title: Connect your SCIM API service to Okta
+excerpt: Create a SCIM integration directly within Okta, configure the SCIM options for that integration, and check the attributes and corresponding mappings.
+meta:
+  - name: description
+    content: Create a SCIM integration directly within Okta, configure the SCIM options for that integration, and check the attributes and corresponding mappings.
+layout: Guides
+---
+
+This guide — part of the [Build a SCIM provisioning integration](/docs/guides/scim-provisioning-integration-overview/) series — teaches you how to create a SCIM integration directly within Okta, configure the SCIM options for that integration, and check the attributes and corresponding mappings.
```

Comment:
This could be shortened a bit:
- There's no need to include "part of the ..." as their already in the guide. It should be clear from context.
- "This guide" is also somewhat redundant as again, they're in the guide.

How about something like:
Learn how to create a SCIM integration, configure the integration's SCIM options, and check the attributes and corresponding mapping using Okta.

Reply:
Done, mostly. I like having the text slightly more conversational, so I've kept "this guide"